src/data/Cameras.jsx

```diff
@@ -0,0 +1,144 @@
+var Cameras = {
+  "all" : [
+    {"ID":"C01","latitude":"-18.9325","longitude":"34.3621","veg_type":"Mixed Savanna and Woodland","dist_water_m":195,"dist_humans_m":1195},
```

Comment:
I wasn't aware that we could `require` JSONs - thanks for that! I'll incorporate that change into the new branch I'm working on.